Lib/WebHandler.py

```diff
@@ -161,8 +163,36 @@ def __init__(self, *args, **kwargs):
     self.__disetConfig.setDecorator(self.__disetBlockDecor)
     self.__disetDump = self.__disetConfig.dump()
     match = self.PATH_RE.match(self.request.path)
-    self._pathResult = self.__checkPath(*match.groups())
+    self.log.info('self.request.path:', self.request.path)
+    self.log.info('self.PATH_RE.pattern:', self.PATH_RE.pattern)
+    self.log.info('match.groups():', match.groups())
+    __pathItems = match.groups()
```

Comment:
please do not use private local variables. I see you are using also other places please fix them too.
Now, we have in this file:
```python
a = None
__a = None
```